tests/wpt/mozilla/tests/mozilla/form_attribute.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
```

Comment:
All of these tests belong in tests/wpt/web-platform-tests/html/semantics/forms/form-control-infrastructure/ since they're testing behaviour that is described in the specification.